src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,46 @@
+package controller;
+
+import view.InputView;
+import view.ResultView;
+import domain.model.LottoNumbers;
+import domain.model.LottoWallet;
+import domain.engine.LottoMachine;
+import domain.generator.RandomLottoNumberGenerator;
+
+import java.util.List;
+
+
+public class LottoController {
+    private final LottoWallet lottoWallet = new LottoWallet();
+    private final LottoMachine lottoMachine = new LottoMachine(new RandomLottoNumberGenerator());
+
+    public void run() {
+        int purchaseAmount = InputView.getPurchaseAmount();
+        purchase(purchaseAmount);
+
+        draw();
+
+        calculateProfit(purchaseAmount);
+    }
+
+    private void purchase(int purchaseAmount) {
+        List<LottoNumbers> lottoList = this.lottoMachine.buyLottos(purchaseAmount);
+        ResultView.printPurchasedLottoCount(lottoList.size());
+        this.lottoWallet.addLottos(lottoList);
+
+        for (LottoNumbers lotto : this.lottoWallet.getLottos()) {
+            ResultView.printLottoNumbers(lotto.getNumbers());
+        }
```

Comment:
`ResultView`를 반복하여 호출하는 대신

`ResultView`에게 `lottoWallet`을 출력해달라고 메시지를 보내보면 어떨까요?

```suggestion
        ResultView.printLottoNumbers(lottoWallet);
```

Reply:
view에 도메인 객체에 대한 의존성이 들어가면 안된다고 생각해서 for문을 돌며 numbers만 넘기는 방식으로 구현했는데요, 반드시 그럴 필요는 없는것인지 질문드리고 싶습니다!

Reply:
뷰에서 도메인 객체를 직접 참조하는 것이 반드시 나쁜 것은 아닙니다.
핵심은 의존성이 단방향으로 흐르는가입니다. 

view는 출력의 책임을 갖고 있기에
비즈니스 로직의 구현을 피하고 출력 로직에만 집중하면 됩니다.

- 로또를 어떻게 표현하지? --> view에게 맡긴다.
  - 로또를 JSON 데이터로 표현하자 --> view에게 맡긴다.
  - 로또를 단순 텍스트로 표현하자 --> view에게 맡긴다.

가령 회원 목록 json 배열을 Front-End에 응답하면 
Front-End에서 for문을 돌려 여러 사용자를 예쁘게 렌더링해 보여주는 것도 Front-End의 책임인 것 처럼요